source/_docs/energy/electricity-grid.markdown

```diff
@@ -59,6 +59,10 @@ _Attention! Installing CT clamp sensor devices requires opening your electrical
 
 Some energy providers will provide you real-time information about your usage and have this data integrated into Home Assistant.
 
+### Troubleshooting ###
+If you are unable to select your energy sensor in the grid consumption drop down, make sure that its value is being recorded in the Recorder settings.
+
```

Comment:
```suggestion
### Troubleshooting ###

If you are unable to select your energy sensor in the grid consumption drop-down, make sure that its value is being recorded in the Recorder settings.
```

Reply:
Besides, I'm not really sure if we should list this on this page even.
This is a general story page, not focussed on the technical configuration at all.

That causes this single troubleshooting item to feel out of place on this page.